route/dht.go

```diff
@@ -0,0 +1,19 @@
+package route
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/thunderdb/ThunderDB/rpc"
+	"net"
+)
+
+// InitDHTserver
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported function InitDHTserver should be of the form "InitDHTserver ..."